pkg/client/openstack/admin/client.go

```diff
@@ -376,6 +378,31 @@ func (c *adminClient) CreateStorageContainer(projectID, containerName, serviceUs
 	return err
 }
 
+func (c *adminClient) ExistsStorageContainer(projectID, containerName string) (bool, error) {
```

Comment:
Maybe `StorageContainerExists` ? The other way around does not roll off the tongue that easily.